CloudFormation/artifactory-enterprise.json

```diff
@@ -902,6 +902,21 @@
                   "         --region ", { "Ref" : "AWS::Region" }, "\n",
                   "runas=root\n"
                 ]]}
+              },
+              "/var/opt/jfrog/artifactory/etc/info/installer-info.json" : {
+                "content" : { "Fn::Join" : ["", [
+                  "{\n",
+                  "  \"productId\": \"JFrogInstaller_Cloudformation_MySQL/1.0.0\"\n",
+                  "  \"features\": [\n",
+                  "  {\n",
+                  "    \"featureId\": \"Artifactory/", { "Ref": "ArtifactoryVersion" }, "\"\n",
+                  "    \"attributes\": {}\n",
```

Comment:
Since attributes are optional and it is empty here, you can skip that.

Reply:
line removed